### PR TITLE
Separate each illuminate\support component to dedicated file

### PR DESCRIPTION
**Pull request by luqmanrom**
-  Separate each illuminate\Support to sub-components
-  Add index file to list and navigate to each of the subcomponents

### CODE REVIEW

File: components/support/index.php [hunk 1]
```diff
@@ -8,106 +8,46 @@
...
-    echo '<h2>Fluent</h2>';
-    echo '<pre>';
-    $personRecord = [
-        'first_name' => 'Mohammad',
-        'last_name' => 'Gufran'
-    ];
-    $record = new Fluent($personRecord);
 
-    $record->address('hometown, street, house');
+$app->get('/collection', function() {
```
[the remark this answers is not in the document]
Comment (mattstauffer) [reply components/support/index.php]: @jasteele12 ¯\(°_o)/¯ that's PSR-2 :)

[the remark this answers is not in the document]
Comment (jasteele12) [reply components/support/index.php]: Oh yeah, the stupid one with spaces instead of tabs too ;-P

**Comment by mattstauffer** [comment]
@luqmanrom Thanks for this PR!

Would you be able to make the changes requested by Nitpick? There's also a @todo left in there.

If so, that'd be awesome, and I'd love to review it again after that.

If not, please let me know. Thanks!

**Comment by AkenRoberts** [comment]
@luqmanrom Did you want to revise this PR as previous requested? Getting stale!

**Pull request closed without merging**